salesforce-pubsub/1.0/modules/ROOT/pages/salesforce-pubsub-connector-message-tracking-example.adoc

```diff
@@ -0,0 +1,73 @@
+= Salesforce Pub/Sub Connector 1.0 - Tracking Salesforce Platform Events Example
+
+One way you can use Salesforce Pub/Sub Connector is to track whether a Salesforce platform event is received from MuleSoft and processed. You can keep track of processed messages using a combination of Salesforce and Mule runtime engine (Mule). 
+
+For a detailed example of one approach to message tracking, see xref:https://docs.google.com/document/d/1NXLE93nzo7IRy4HSHsxh_InhhDJGv_m_xGo91VDyb3U/edit#heading=h.3xat2b92h1uu[Keeping Track of the Processed and Received Messages]
+
+== Replay ID and MuleSoft Object Store
+
+To implement message tracking, it's important to understand the core concepts of the Salesforce Streaming API, the Replay ID attribute, and MuleSoft object store.
+
+[[replay-id]]
+=== Replay ID
+
+On the Salesforce Event Bus, each published event message is assigned an opaque ID, which is contained in the Replay (`replay_id`) attribute. The Replay ID field value is populated by the system when the event message is delivered to subscribers and refers to the position of the event message within the event stream. 
+
+The event stream has a message durability of 72 hours for high-volume platform events so, using the Replay ID, it's possible to receive event messages from the event stream that fall within that period. The MuleSoft subscriber must carefully track which Replay ID is processed to ensure it doesn't miss or duplicate process events in case of down time or restart (either intentional or unintentional). 
```

Comment:
```suggestion
The event stream has a message durability of 72 hours for high-volume platform events so, using the Replay ID, it's possible to receive event messages from the event stream that fall within that period. The MuleSoft subscriber must carefully track the processed Replay IDs to ensure it doesn't miss or duplicate process events in case of downtime or restart (either intentional or unintentional). 
```